finals-2024-01-16/code/5238-mert-miftar/src/app/band-list/band-list.component.html

```diff
@@ -1 +1,43 @@
-<p>And the band list and everything else comes here</p>
+<button (click)="statistics()">Statistics</button> 
```

Comment:
+2 points for header functionality - however, it's better if that is a separate component